<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wardrobeDialog</class>
 <widget class="QDialog" name="wardrobeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>144</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>27</pointsize>
    </font>
   </property>
   <property name="text">
    <string>衣柜查询</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>描述更多关于它的故事 : </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>160</width>
     <height>220</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>211</width>
     <height>248</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>名称 :     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_5">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>类别 :     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>衣服</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>饰品</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>包包</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>化妆品</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>鞋帽</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>季节 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>春季</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>夏季</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>秋季</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>冬季</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>分类 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>职场</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>休闲</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>运动</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>宴会</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>其他</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>品牌 :     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>价格 :     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>时间 : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>存放地 :   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>购买地 :   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>275</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>存放地 :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_4">
      <property name="minimumSize">
       <size>
        <width>135</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>135</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>共计 : 1234件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
